app/graphql/types/activity_type.rb

```diff
@@ -0,0 +1,7 @@
+module Types 
+	class ActivityType < Types::BaseObject 
+		field :id, ID, null: false
+		field :name, String, null: false
+		field :est_time, Integer, null: false
```

Comment:
This looks great. Will this be affected by the Integer to Interval change of Becky's?

Reply:
This will not be affected as far as I can tell. Becky and I spoke about it before I PR'd and time will be added in and sent to FE, but is not currently a requirement for a user to input.